src/systems/a320_systems/src/hydraulic.rs

```diff
@@ -1,20 +1,3667 @@
+            && (self.left_brake_pilot_input < 0.2 && self.right_brake_pilot_input < 0.2)
+        {
+            self.normal_brakes_available = true;
+        } else if normal_braking_loop_pressure.get::<psi>() < Self::MIN_PRESSURE_BRAKE_ALTN_HYST_LO
+        {
+            self.normal_brakes_available = false;
+        }
+    }
+
+    fn update_brake_pressure_limitation(
+        &mut self,
+        norm_brk: &mut BrakeCircuit,
+        altn_brk: &mut BrakeCircuit,
+    ) {
+        let yellow_manual_braking_input = self.left_brake_pilot_input
+            > self.left_brake_yellow_output + 0.2
+            || self.right_brake_pilot_input > self.right_brake_yellow_output + 0.2;
+
+        // Nominal braking from pedals is limited to 2538psi
+        norm_brk.set_brake_limit_active(true);
+        norm_brk.set_brake_press_limit(Pressure::new::<psi>(2538.));
+
+        if self.parking_brake_demand {
+            altn_brk.set_brake_limit_active(true);
+
+            // If no pilot action, standard park brake pressure limit
+            if !yellow_manual_braking_input {
+                altn_brk.set_brake_press_limit(Pressure::new::<psi>(2103.));
+            } else {
+                // Else manual action limited to a higher max nominal pressure
+                altn_brk.set_brake_press_limit(Pressure::new::<psi>(2538.));
+            }
+        } else if !self.anti_skid_activated {
+            altn_brk.set_brake_press_limit(Pressure::new::<psi>(1160.));
+            altn_brk.set_brake_limit_active(true);
+        } else {
+            // Else if any manual braking we use standard limit
+            altn_brk.set_brake_press_limit(Pressure::new::<psi>(2538.));
+            altn_brk.set_brake_limit_active(true);
+        }
+    }
+
+    /// Updates final brake demands per hydraulic loop based on pilot pedal demands
+    fn update_brake_demands(
+        &mut self,
+        context: &UpdateContext,
+        green_loop: &HydraulicLoop,
+        alternate_circuit: &BrakeCircuit,
+        landing_gear: &LandingGear,
+    ) {
+        self.update_normal_braking_availability(&green_loop.pressure());
+
+        let is_in_flight_gear_lever_up = !self.weight_on_wheels && !self.is_gear_lever_down;
+        self.should_disable_auto_brake_when_retracting.update(
+            context,
+            !landing_gear.is_down_and_locked() && !self.is_gear_lever_down,
+        );
+
+        if is_in_flight_gear_lever_up {
+            if self.should_disable_auto_brake_when_retracting.output() {
+                self.left_brake_green_output = 0.;
+                self.right_brake_green_output = 0.;
+                self.left_brake_yellow_output = 0.;
+                self.right_brake_yellow_output = 0.;
+            } else {
+                // Slight brake pressure to stop the spinning wheels
+                self.left_brake_green_output = 0.2;
+                self.right_brake_green_output = 0.2;
+            }
+        } else {
```

Comment:
**suggestion**: `..._yellow_output` is never reset, which might be an error, consider:
```suggestion
        if is_in_flight_gear_lever_up {
            if self.should_disable_auto_brake_when_retracting.output() {
                self.left_brake_green_output = 0.;
                self.right_brake_green_output = 0.;
            } else {
                // Slight brake pressure to stop the spinning wheels
                self.left_brake_green_output = 0.2;
                self.right_brake_green_output = 0.2;
            }
            self.left_brake_yellow_output = 0.;
            self.right_brake_yellow_output = 0.;
        } else {
```

Reply:
Thought it was already done, now it's done!